x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/model/rule_schema/specific_attributes/eql_attributes.schema.yaml

```
@@ -10,7 +10,7 @@ components:
       type: string
     TimestampField:
       type: string
-      description: Contains the event timestamp used for sorting a sequence of events
+      description: Contains the event timestamp used for sorting a sequence of events. This is different from timestamp_override, which is used for querying events within a range. Defaults to the @timestamp ECS field.
```

Nit: "contains" is too vague of a verb; what else does `TimestampField` contain?

```suggestion
      description: Specifies the name of the event timestamp field used for sorting a sequence of events. Not to be confused with `timestamp_override`, which specifies the more general field used for querying events within a range. Defaults to the @timestamp ECS field.
```

---

I agree, your version sounds better then what I copied from legacy docs. Applying the change.